### PR TITLE
update overridden config for 1.10

### PR DESCRIPTION
**Pull request by Bradamant3**
Oops. Suspect this happened a while back. Fixed before release anyway ...

/assign @zacharysarah

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -19,16 +19,21 @@ defaults:
     scope:
       path: ""
     values:
-      fullversion: "v1.9.0"
-      version: "v1.9"
+      fullversion: "v1.10.0"
```
Comment (tengqm) [inline _config.yml]: not sure if line 16 should be changed as well

Comment (zhangxiaoyu-zidif) [reply _config.yml]: yes, it should be changed as well, if it was in Release 1.10

**Comment by Bradamant3** [comment]
Thanks folks. Why I love PRs and reviews!

**Pull request merged**